demo/android/rust/src/lib.rs

```diff
@@ -167,6 +170,28 @@ impl Window for WindowImpl {
         Point2DI32::new(0, 0)
     }
 
+    fn view_box_size(&self, mode: Mode) -> Point2DI32 {
```

Comment:
How about something like:

    fn viewport(&self, view: View) -> Rect2DI32 {
        match view {
            View::Mono => { ... }
            View::Stereo(eye) => { ... }
    }

See below for rationale.